<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="welcome">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="balance">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="charge">
     <property name="text">
      <string>Charge balance</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="myOrders">
     <property name="text">
      <string>My orders</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="makeOrder">
     <property name="text">
      <string>Make order</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="logOut">
     <property name="text">
      <string>Log out</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
